template/CHART_NAME/templates/daemonset.yaml

```diff
@@ -0,0 +1,126 @@
+  {{- end }}
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: %%COMPONENT_NAME%%
+  template:
+    metadata:
+      {{- if .Values.%%MAIN_OBJECT_BLOCK%%.podAnnotations }}
+      annotations: {{- include "common.tplvalues.render" (dict "value" .Values.%%MAIN_OBJECT_BLOCK%%.podAnnotations "context" $) | nindent 8 }}
+      {{- end }}
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: %%COMPONENT_NAME%%
+        {{- if .Values.%%MAIN_OBJECT_BLOCK%%.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.%%MAIN_OBJECT_BLOCK%%.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+    spec:
+      serviceAccountName: {{ template "%%TEMPLATE_NAME%%.serviceAccountName" . }}
+      {{- include "%%TEMPLATE_NAME%%.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.%%MAIN_OBJECT_BLOCK%%.affinity }}
+      affinity: {{- include "common.tplvalues.render" ( dict "value" .Values.%%MAIN_OBJECT_BLOCK%%.affinity "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.%%MAIN_OBJECT_BLOCK%%.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.%%MAIN_OBJECT_BLOCK%%.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.%%MAIN_OBJECT_BLOCK%%.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.%%MAIN_OBJECT_BLOCK%%.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.%%MAIN_OBJECT_BLOCK%%.priorityClassName }}
+      priorityClassName: {{ .Values.%%MAIN_OBJECT_BLOCK%%.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.%%MAIN_OBJECT_BLOCK%%.podSecurityContext.enabled }}
+      securityContext:
+        fsGroup: {{ .Values.%%MAIN_OBJECT_BLOCK%%.podSecurityContext.fsGroup }}
+      {{- end }}
+      {{- if .Values.%%MAIN_OBJECT_BLOCK%%.initContainers }}
+      initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.%%MAIN_OBJECT_BLOCK%%.initContainers "context" $) | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: %%CONTAINER_NAME%%
+          image: {{ template "%%TEMPLATE_NAME%%.image" . }}
+          imagePullPolicy: {{ .Values.%%MAIN_OBJECT_BLOCK%%.image.pullPolicy }}
+          {{- if .Values.%%MAIN_OBJECT_BLOCK%%.lifecycleHooks }}
+          lifecycle: {{- include "common.tplvalues.render" (dict "value" .Values.%%MAIN_OBJECT_BLOCK%%.lifecycleHooks "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.%%MAIN_OBJECT_BLOCK%%.containerSecurityContext.enabled }}
+          securityContext:
+            runAsUser: {{ .Values.%%MAIN_OBJECT_BLOCK%%.containerSecurityContext.runAsUser }}
+            runAsNonRoot: {{ .Values.%%MAIN_OBJECT_BLOCK%%.containerSecurityContext.runAsNonRoot }}
+          {{- end }}
+          command:
```

Comment:
I would rather leave command not set by default, we would normally expect to use the container default command and use the `command` value if set